packages/synapse-react-client/src/mocks/msw/handlers/asyncJobHandlers.ts

```diff
@@ -119,7 +128,10 @@ export function generateAsyncJobHandlers<
         const responseObject: TResponseBody =
           typeof response === 'function' ? response(request) : response
 
-        return res(ctx.status(201), ctx.json(responseObject))
+        return res(
+          ctx.status(serviceSpecificEndpointResponseStatus),
```

Comment:
We call this endpoint to get the status code and figure out the type of error.

Our SynapseClient code calls this endpoint because if the job failed, the other service doesn't give us the information we need to determine if we should retry or not. This endpoint gives us an HTTP status code, which we can use to decide to retry.